### PR TITLE
Update docs to reflect namespaces

### PR DESCRIPTION
**Pull request by minasmart**
Note to reviewers: Just try to comb over this sorta carefully. I think I updated all the references, but could have missed a few.

### CODE REVIEW

**Comment by chrisbutcher** [comment]
@minasmart I think you made us assignees, not reviewers :P

File: src/collection-resource.js [hunk 1]
```diff
@@ -104,18 +102,16 @@ export default class CollectionResource extends Resource {
...
-   *   @param {String} [queryObject.sortBy] The field to use to sort collections. Possible values are `title` and `updatedAt`.
-   *   @param {String} [queryObject.sortDirection] The sort direction of the collections.
-   *     Will sort collections by ascending order unless `'desc'` is specified.
-   * @param {Client.Queries.collectionConnectionQuery} [query] Callback function to specify fields to query on the collections.
+   * @param {Object} [args] An object specifying the query data containing zero or more of:
+   *   @param {Int} [args.first=20] The relay `first` param. This specifies page size.
+   *   @param {String} [args.sortKey=ID] The key to sort results by. Available values are
+   *   documented as {@link https://help.shopify.com/api/storefront-api/reference/enum/collectionsortkeys|Collection Sort Keys}.
+   *   @param {String} [args.query] A query string. See full documentation {@link https://help.shopify.com/api/storefront-api/reference/object/shop#collections|here}
+   *   @param {Boolean} [args.reverse] Wether or not to reverse the sort order of the results
```
Comment (harisaurus) [inline src/collection-resource.js]: `whether`

File: src/product-resource.js [hunk 2]
```diff
@@ -72,20 +71,16 @@ export default class ProductResource extends Resource {
    * Fetches all products on the shop that match the query.
    *
    * @example
-   * client.fetchQueryProducts({sortBy: 'title', limit: 10}).then((products) => {
+   * client.product.fetchQuery({first: 20, sortKey: "CREATED_AT", reverse: true}).then((products) => {
```
Comment (harisaurus) [inline src/product-resource.js]: Single quotes?

File: src/product-resource.js [hunk 3]
```diff
@@ -72,20 +71,16 @@ export default class ProductResource extends Resource {
...
-   *   @param {String} [queryObject.sortBy] The field to use to sort products. Possible values are `title`, `updatedAt`, and `createdAt`.
-   *   @param {String} [queryObject.sortDirection] The sort direction of the products.
-   *     Will sort products by ascending order unless `'desc'` is specified.
-   * @param {Client.Queries.productConnectionQuery} [query] Callback function to specify fields to query on the products.
+   * @param {Object} [args] An object specifying the query data containing zero or more of:
+   *   @param {Int} [args.first=20] The relay `first` param. This specifies page size.
+   *   @param {String} [args.sortKey=ID] The key to sort results by. Available values are
+   *   documented as {@link https://help.shopify.com/api/storefront-api/reference/enum/productsortkeys|Product Sort Keys}.
+   *   @param {String} [args.query] A query string. See full documentation {@link https://help.shopify.com/api/storefront-api/reference/object/shop#products|here}
+   *   @param {Boolean} [args.reverse] Wether or not to reverse the sort order of the results
```
Comment (harisaurus) [inline src/product-resource.js]: `Whether`

Comment (chrisbutcher) [reply src/product-resource.js]: image

🐮 🚫 ✂️

**Pull request merged**